# Request
sensor_msgs/PointCloud2 input_cloud
pcl_msgs/PointIndices target_indices
---
# Response
sensor_msgs/PointCloud2 filtered_cloud